opencog/nlp/learn/README

```diff
@@ -141,30 +144,44 @@ So, set up the text-ingestion pipeline:
    (add-to-load-path "/usr/local/share/opencog/scm")
    (add-to-load-path ".")
 ```
-
-6) Select a language, and copy one of the `pair-count-??.scm` files
-   from the `run` directory to your working directory. Review and edit
-   the configuration as necessary. It contains the REPL port number,
-   and a default prompt: These are set up by default to avoid conflicts
-   and confusion, and to allow multiple languages to be processed at
-   the same time.  It also contains the database credentials -- that's
-   probably the only thing you need to change.
-
-7) Start the REPL server. Eventually, you can use the
-   `run-all-servers.sh` file in the `run` directory to do this;
+4) Copy the following files from the `opencog/opencog/nlp/learn/run` directory
+   to your working directory:
+
+   - run-server-parse.sh
+   - observe-launch.scm
+   - opencog-en.conf
+   - utilities.scm
+   - wiki-ss-en.sh
+   - ss-one.sh
+   - submit-one.pl
+   - nonbreaking_prefixes (the complete folder)
+
+   `run-server-parse.sh` starts the cogserver and sets a default prompt: set 
+   up by default to avoid conflicts and confusion, and to allow multiple 
+   languages to be processed at the same time. 
+   You need to pass the language and database credentials as arguments; 
+   see next step.
+   `wiki-ss-en.scm` starts the process of observing the text files contained
+   in the beta-pages folder, more on that later (see Bulk Text Parsing).
+   The rest of the files are explained later.
+
+5) Start the REPL server. Eventually, you can use the
+   `run-server-parse.sh` file in the `run` directory to do this;
```

Comment:
There is no `run-server-parse.sh` file!

Reply:
Yes, it is: https://github.com/opencog/opencog/blob/master/opencog/nlp/learn/run/run-server-parse.sh